src/tests/ftest/process_core_files.py

```diff
@@ -101,8 +106,12 @@ def process_core_files(self, directory, delete):
                         command = ["lbzip2", "-d", "-v", os.path.join(core_dir, core_name)]
                         run_local(self.log, " ".join(command))
                         core_name = os.path.splitext(core_name)[0]
-                    self._create_stacktrace(core_dir, core_name)
-                    corefiles_processed += 1
+                    exe_name = self._get_exe_name(os.path.join(core_dir, core_name))
+                    self._create_stacktrace(core_dir, core_name, exe_name)
+                    if test in CORE_FILES_IGNORE and exe_name in CORE_FILES_IGNORE[test]:
+                        self.log.info('core file name configured for ignore, skipping')
```

Comment:
It would be nice to log the details, but that could be added later.:
```suggestion
                        self.log.debug('Excluding %s core file (%s) detected while running %s from processed core count', core_name, exe_name, test)
```

Reply:
@mjean308 when this lands and we have this capability in process_core_files.py, it may also make sense to replace the launch.py `TEST_EXPECT_CORE_FILES = ["./harness/core_files.py"]` code with a `CORE_FILES_IGNORE ` entry for the `./harness/core_files.py` test.

Reply:
I was thinking the same Phil.

Reply:
will do... I will create a ticket for this